gammapy/background/cube.py

```diff
@@ -809,6 +809,38 @@ def plot_spectrum(self, coord, ax=None, style_kwargs=None):
             plt.close(fig)
         return ax
 
+    @property
+    def integral(self):
+        """Integral of the cube (`~astropy.units.Quantity`)"""
```

Comment:
What is the "Integral of the cube"?

I think it's predicted counts?

If so you could call it `npred` or `npred_total` (see 
https://gammapy.readthedocs.org/en/latest/api/gammapy.data.SpectralCube.html#gammapy.data.SpectralCube.integral_flux_image and
https://gammapy.readthedocs.org/en/latest/api/gammapy.data.compute_npred_cube.html),
or at least mention this in the docstring.

Also maybe add "scalar" to make it clearer?
```
(scalar `~astropy.units.Quantity`)
```